vi_cleaner/abbreviation_vi.py

```diff
@@ -15,26 +15,18 @@
     "bik": "biết",
 }
 
-_abbreviations_combine_re = r"(" + "|".join(_abbreviations_vi.keys()) + r")"
-
 def _expand_percent_vi(m):
     return " phần trăm"
 
 
 def _expand_urls_vi(m):
     return f"{m.group(1)} chấm {m.group(2)}"
 
-
-def _expand_abbreviations_vi(m):
-    key = m.group(0)
-    key = key.replace(".", "\.").lower()
-    return _abbreviations_vi[key]
-
-
 def normalize_abbreviations_vi(text):
     text = normalize_speacial_symbol_vi(text)
     text = re.sub(_url_re, _expand_urls_vi, text)
-    text = re.sub(_abbreviations_combine_re, _expand_abbreviations_vi, text, flags=re.IGNORECASE)
+    for k, v in _abbreviations_vi.items():
+        text = re.sub(r"\b" + k + r"\b", v, text, flags=re.IGNORECASE)
```

Comment:
Maybe we can somehow don't use for loop

Reply:
Hmm It was not in the past but then the error persists i.e: kounde -> khôngunde

Reply:
I tried to but the r"\b" between then re but to no avail, it got processed like this ("\bko\b"|"\bco\b"|")

Reply:
Maybe you can try adding `_vietnamese-re` (which is in file symbol_vi.py) to verify the abbr is not in the middle of another word.

Reply:
done em

Reply:
Don't know if it works.

Reply:
Hi em, I think this is good enough ?
<img width="1120" alt="image" src="https://user-images.githubusercontent.com/101542319/181416311-2b2cd0ad-eed6-4035-ab09-b5a41e64482a.png">

Reply:
Cool, LGTM!